### PR TITLE
make forloop without useless index

### PR DESCRIPTION
**Pull request by mjaow**
Use for loop with item (not index)

---
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1241)

### CODE REVIEW

File: pkg/executor/poolmgr/gpm.go [hunk 1]
```diff
@@ -267,11 +267,11 @@ func (gpm *GenericPoolManager) eagerPoolCreator() {
 		// creating pools for envs that are actually used by functions.  Also we might want
 		// to keep these eagerly created pools smaller than the ones created when there are
 		// actual function calls.
-		for i := range envs.Items {
-			env := envs.Items[i]
+		for _, item := range envs.Items {
```
Comment (vishal-biyani) [inline pkg/executor/poolmgr/gpm.go]: We could directly get env in the loop:

```
    for _, env := range envs.Items {
```

Comment (mjaow) [reply pkg/executor/poolmgr/gpm.go]: right,i'm done

**Comment by life1347** [comment]
No, we cannot remove the index here. This is in intentional for doing that. As the pointer of the environment is the address to the loop variable, however, it's value will be changed as loop iterate to next item.

**Comment by life1347** [comment]
For details, see https://github.com/fission/fission/pull/221

**Comment by life1347** [comment]
Close this as the problem mentioned above. But @mjaow thanks for your PR!

**Pull request closed without merging**